src/navigation/Nav.jsx

```diff
@@ -282,7 +283,11 @@ export class Nav extends React.Component {
 		};
 
 		const getConnectionsIcon = () => (
-			<img className="proIcon" alt={connections.label} src={CONNECTIONS_ICON} />
+			<img
+				className="proIcon"
+				alt={connections && connections.label}
```

Comment:
Is it possible that connections can be `null` or `undefined`?  If so let's have a fallback string value here

Reply:
yep it's possible cause on pro-web connections not implemented and I just try to fix issue for update a lib version, it's not a part of my work

Reply:
I think it should be removed when connections will be implemented on pro-web, but idk who deals with this

Reply:
I see, but it will fix the type error, but still will leave alt attribute blank. Why not to use ternary here? `connectios ? connections.label : 'pro-icon` (or some other fallback)